instant-distance-py/examples/perf-fasttext.rs

```diff
@@ -0,0 +1,111 @@
+//! This is a simple example of how to use the instant-distance crate to build an index from a file.
+//! The file is expected to be in the format of the fasttext word vectors.
+//!
+//! This example was built to performance using such tools as valgrind, perf, and heaptrack.
+//!
+//! Get the required file:
+//! wget https://dl.fbaipublicfiles.com/fasttext/vectors-aligned/wiki.en.align.vec
+
+use std::fs::File;
+use std::io::{BufRead, BufReader};
+use std::path::{Path, PathBuf};
+use std::thread::sleep;
+use std::time::{Duration, Instant};
+
+use clap::Parser;
+use instant_distance::{Builder, PointDataSource, PointRef, Search};
+use instant_distance_py::FloatArray;
+
+#[global_allocator]
+static ALLOC: tracy_full::alloc::GlobalAllocator = tracy_full::alloc::GlobalAllocator::new();
+
+#[derive(Parser)]
+struct Opt {
```

Comment:
Per the convention of top-down ordering, I'd prefer to call this `Options` and move it below `main()`. `ALLOC` should be defined at the bottom of the module.

Reply:
Done